src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -33,54 +89,28 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
-          Transaction hash:{' '}
-        </Text>
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
+      {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" />}
+      {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
+      <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
+      <br></br>
+
+      {/* Advanced TxData */}
+      {txData?.operation !== undefined && (
+        <TxDataRow title="Operation:" value={`${txData?.operation} (${Operation[txData?.operation].toLowerCase()})`} />
       )}
-      {created && (
-        <div className="tx-created">
-          <Text size="xl" strong as="span">
-            Created:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {formatDateTime(created)}
-          </Text>
-        </div>
+      {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
+      {baseGas && <TxDataRow title="baseGas:" value={baseGas} />}
+      {gasPrice && <TxDataRow title="gasPrice:" value={gasPrice} />}
+      {gasToken && <TxDataRow title="gasToken:" value={gasToken} inlineType="hash" />}
+      {refundReceiver && <TxDataRow title="refundReceiver:" value={refundReceiver} inlineType="hash" />}
+      {confirmations?.map(({ signature }, index) => (
+        <TxDataRow title={`Signature ${index + 1} :`} value={signature} inlineType="rawData" key={index} />
+      ))}
+      {confirmations?.length > 1 && (
+        <TxDataRow title="Signatures :" value={signaturesFromConfirmations} inlineType="rawData" />
       )}
-      <div className="tx-executed">
-        <Text size="xl" strong as="span">
-          Executed:{' '}
-        </Text>
-        <Text size="xl" as="span">
-          {executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE}
-        </Text>
-      </div>
+      {txData?.hexData && <TxDataRow title="Raw data:" value={txData?.hexData} inlineType="rawData" />}
       {txData?.operation === Operation.DELEGATE && (
         <div className="tx-operation">
           <DelegateCallWarning isKnown={isCustomTxInfo(txInfo) && !!txInfo?.to?.name} />
```

Comment:
Does the delegate warning fit with the new style adjustments? It is also present on nested delegate calls.

Reply:
It seems to be right. Might be edges cases that I didn't test
<img width="775" alt="Screen Shot 2021-12-17 at 14 53 57" src="https://user-images.githubusercontent.com/32431609/146554782-5a68639b-8cdf-4a40-af02-01932dff1375.png">